common/lib/xmodule/xmodule/capa_base.py

```diff
@@ -1423,7 +1423,9 @@ def save_problem(self, data):
         self.track_function_unmask('save_problem_success', event_info)
         msg = _("Your answers have been saved.")
         if not self.max_attempts == 0:
-            msg = _("Your answers have been saved but not graded. Click 'Check' to grade them.")
+            msg = _(
+                    "Your answers have been saved but not graded. Click {check} to grade them."
```

Comment:
I would recommend calling this {button_name}. Translators don't get any of the context, they just get the string itself, and {button_name} is the most descriptive way to describe what exactly is being defined here.